Recipely/Recipely/NetworkService/RequestCreator.swift

```diff
@@ -0,0 +1,104 @@
+// RequestCreator.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Перечилсение типов
+enum DishType: String {
+    /// Тип ячейки с салатами
+    case salad
+    /// Тип ячейки с садлатами
+    case soup
+    /// Тип ячейки с пирожными
+    case pancake
+    /// Тип ячейки с напитками
+    case drinks
+    /// Тип ячейки с десертами
+    case desserts
+    /// Тип ячейки с курицей, с мясом, с рыбой с sideDish
+    case chicken, meat, fish, sideDish
+
+    var dishCategory: String {
+        switch self {
+        case .chicken, .meat, .fish, .sideDish:
+            return "Main Course"
+        case .salad:
+            return "salad"
+        case .soup:
+            return "soup"
+        case .pancake:
+            return "pancake"
+        case .drinks:
+            return "drinks"
+        case .desserts:
+            return "desserts"
+        }
+    }
+}
+
+/// Варианты path ссылки
+enum PuthState: String {
```

Comment:
```suggestion
enum PuthState: String {
```
эти энамы не могут быть внутри реквестера?